CESC-submission/paper.tex

```diff
@@ -62,15 +62,15 @@
 
 
 \begin{abstract}
-Escalating demand for digital services has motivated building non-traditional solutions to reduce costs and achieve a greater level of flexibility. Among these solutions, peer-assisted models are gaining more popularity in replacing infrastructure-based (possibly) centralized approaches. Cryptocurrencies have strengthened this trend by providing a fully distributed mechanism to reward for the service, and their blockchains and consensus protocols offer a trustless and publicly verifiable way to govern the system. We present a generic framework for building distributed service markets by utilizing these new technologies. We also discuss the security and efficiency challenges, with a focus on how the economic aspects impact system design and threat mitigation, and conclude with some potential solutions.
+Escalating demand for digital services has motivated building non-traditional solutions to reduce costs and achieve a greater level of flexibility. Among these solutions, peer-assisted models are gaining more popularity in replacing infrastructure-based (possibly) centralized approaches. Cryptocurrencies have strengthened this trend by providing a fully distributed mechanism to compensate service provision, and their blockchains and consensus protocols offer a trustless and publicly verifiable way to govern the system. We present a generic framework for building distributed service markets by utilizing these new technologies. We also discuss the security and efficiency challenges, with a focus on how the economic considerations impact system design and threat mitigation, and conclude with some potential solutions.
```

Comment:
reward for the service -> compensate service provision
aspects -> considerations